src/shared/codelens/localLambdaRunner.ts

```diff
@@ -705,3 +713,20 @@ function getAttachDebuggerMaxRetryLimit(
         defaultValue
     )!
 }
+
+/**
+ * Brings the Debug Console in focus.
+ * If the OutputChannel is showing, focus does not consistently switch over to the debug console, so we're
+ * helping make this happen.
+ */
+async function showDebugConsole({
+    executeVsCodeCommand = vscode.commands.executeCommand
+}: {
+    executeVsCodeCommand?: typeof vscode.commands.executeCommand
+}): Promise<void> {
+    try {
+        await executeVsCodeCommand('workbench.debug.action.toggleRepl')
+    } catch (err) {
+        // in case the vs code command changes or misbehaves, swallow error
```

Comment:
Would it make sense to (verbose) log this?

Reply:
Maybe instead, considering popping up an info message. If we swallow/log the error, the user won't know that 1: something bad happened, 2: that output is being generated, and 3: that they should switch to the Debug Console to see the output.

Reply:
Keeping the failure state silent is deliberate in this case. If the command behavior changes on us, it is better not to be inundating the user with messages that are out of their control.